packages/grid/test/drag-and-drop.common.js

```diff
@@ -290,6 +290,23 @@ describe('drag and drop', () => {
         }
       });
 
+      it('should add items to draggedItems array when drag starts', () => {
+        grid.selectItem(grid.items[0]);
+        grid.selectItem(grid.items[1]);
+        fireDragStart();
+
+        expect(grid.__draggedItems.length).to.equal(2);
+      });
+
+      it('should remove items from draggedItems array when drag ends', () => {
+        grid.selectItem(grid.items[0]);
+        fireDragStart();
+
+        expect(grid.__draggedItems.length).to.equal(1);
+
+        fireDragEnd();
+        expect(grid.__draggedItems.length).to.equal(0);
+      });
```

Comment:
These tests are checking `grid.__draggedItems` which is an internal technical detail. Instead, add a test which:
- starts dragging items
- checks that the part name is there on the cell element
- scrolls the grid
- checks that the part name is no longer there

...so kind of like the combination of these tests and the one added to `styling-common.js` but without the tests knowing about the private `__draggedItems` array.

Reply:
I thought about something like that, but couldn't do it because the common tests don't have enough elements for scrolling and the styling tests don't have access to `fireDragStart`. I suppose I need to either export the `fireDragStart` and `fireDragEnd` methods, or add more items to the test. What would be preferential?

Reply:
The issue occurs with the auto-scroll feature of grid DnD, so the test could be added to the "auto scroll" suite of `drag-and-drop.common.js`, which has a grid setup with 100 items.